backend/de.metas.acct.webui/src/main/java/de/metas/acct/gljournal_sap/quickinput/SAPGLJournalLineQuickInputDescriptorFactory.java

```diff
@@ -152,6 +159,33 @@ private LookupDescriptorProvider getPostingSignLookup()
 		return lookupDescriptorProviders.listByAD_Reference_Value_ID(postingSignReferenceId);
 	}
 
+	@NonNull
+	private LookupDescriptorProvider getAccountProvider()
+	{
+		final POInfo poInfo = POInfo.getPOInfo(I_SAP_GLJournalLine.Table_Name);
+		if(poInfo == null){return getAccountProviderFallback();}
+
+		final POInfoColumn columnInfo = poInfo.getColumn(I_SAP_GLJournalLine.COLUMNNAME_C_ValidCombination_ID);
+		if(columnInfo == null){return getAccountProviderFallback();}
+
+		final AdValRuleId valueRuleId = columnInfo.getAD_Val_Rule_ID();
+		if(valueRuleId == null){return getAccountProviderFallback();}
+
+		final SqlLookupDescriptorProviderBuilder descriptorProviderBuilder = lookupDescriptorProviders.sql()
+				.setCtxTableName(I_C_ValidCombination.Table_Name)
```

Comment:
ctxtablename is I_SAP_GLJournalLine.Table_Name
and ctxcolumname is I_SAP_GLJournalLine.COLUMNNAME_C_ValidCombination_ID